documentation/modules/exploit/linux/http/glinet_unauth_rce_cve_2023_50445.md

```diff
@@ -0,0 +1,314 @@
+## Vulnerable Application
+A command injection vulnerability exists in multiple GL.iNet network products, allowing an attacker to inject and execute
+arbitrary shell commands via JSON parameters at the `gl_system_log` and `gl_crash_log` interface in the `logread` module.
+This exploit requires post-authentication using the `AdminToken` cookie / session ID (`SID`), typically stolen by the attacker.
+
+However, by chaining this exploit with vulnerability CVE-2023-50919, one can bypass the Nginx authentication through a `Lua`
+string pattern matching and SQL injection vulnerability.
+The `AdminToken` cookie / `SID` can be retrieved without knowing a valid username and password.
+
+The following GL.iNet network products are vulnerable:
+ - A1300, AX1800, AXT1800, MT3000, MT2500/MT2500A => v4.0.0 < v4.5.0;
+ - MT6000 => v4.5.0 - v4.5.3;
+ - MT1300, MT300N-V2, AR750S, AR750, AR300M, AP1300, B1300 => v4.3.7;
+ - E750/E750V2, MV1000 => v4.3.8;
+ - and potentially others (just try ;-)
```

Comment:
```suggestion
 - A1300, AX1800, AXT1800, MT3000, MT2500/MT2500A: v4.0.0 < v4.5.0;
 - MT6000: v4.5.0 - v4.5.3;
 - MT1300, MT300N-V2, AR750S, AR750, AR300M, AP1300, B1300: v4.3.7;
 - E750/E750V2, MV1000: v4.3.8;
 - and potentially others (just try ;-)
```

This confused me I thought E750/E750V2, MV1000 => v4.3.8" meant "E750/E750V2, MV1000 with a version equal or superior to v4.3.8"

Reply:
yes, agree. see https://github.com/rapid7/metasploit-framework/pull/18648/commits/9bb7e0e37930022e97f801e22ad8dd1f864b061b